datacenter/ucp/2.0/guides/monitor/index.md

```diff
@@ -65,6 +65,10 @@ If you're accessing this endpoint through a load balancer, you have no way to
 know which UCP manager node is not healthy. So make sure you make a request
 directly to each manager node.
 
+## Monitoring disk usage
+
+Web UI disk usage metrics, including free space, only reflect the Docker managed portion of the filesystem: `/var/lib/docker`.  To monitor the total space available on each filesystem of a UCP worker or manager, you must deploy a monitoring solution, such as  Prometheus Node Exporter.
```

Comment:
I'm not sure what the web UI looks like in UCP 2.0 and whether this applies or not.